README.md

```diff
@@ -52,6 +51,76 @@ To reload the extension after some changes, open [`chrome://extensions`](chrome:
 
 To use the local API instead of the prod one, change `ORBIT_ROOT_URL` in `orbit-helper.js` to your ngrok tunnel address.
 
+## Contributing
+
+Pull requests & issues welcome!
+
+### Accessibility
+
+Accessibility is a fundamental part of web design, and something we consider critical to the further development of this extension. Future contirbutions should aim to meet [WCAG2.1 AA](https://www.w3.org/TR/WCAG21/)—if you notice any areas where we're failing, please raise them as issues or submit PRs to address them.
+
+Common issues to check:
+
+1. Verify all functionality is usable with keyboard-only control
+2. Open your screenreader ([VoiceOver](https://support.apple.com/en-gb/guide/voiceover/vo2682/mac) is built-in for macOS, or [Narrator](https://support.microsoft.com/en-us/windows/chapter-1-introducing-narrator-7fe8fd72-541f-4536-7658-bfc37ddaf9c6#ID0EDDBBDBD) in Windows) and confirm navigating across the widget provides clear, contextual, and useful information akin to what is shown visually.
+3. Confirm colours satisfy contrast requirmeents using a tool such as [colourcontrast.cc](https://colourcontrast.cc/)
+
+### Code overview
+
+The extension's functionality mainly relies on two views:
+
+1. **Options View** - Located in the `src/options/` directory. This view displays the page users see when they access the extension's options using the specified chrome-extension URL. Here, you'll find both the markup and behavior logic for the options page.
+
+2. **Widget View** - Found in `src/components/widget.js`, this view represents the popup that appears when users open the extension on a supported page they're browsing.
+
+If you're new to developing browser extensions, there are a couple of crucial parts you need to be aware of:
+
+1. **Manifest File** - Located at `src/manifest.json`. This file instructs the browser when to run which scripts (for example, "on a page that matches a GitHub pull request, run the github entrypoint"). If you plan to extend our extension's capabilities to new sites, you'll have to update this file accordingly.
+
+2. **Background Runner** - The `src/background.js` file handles requests to the Orbit API in a separate thread, ensuring smooth performance. Look for instances of `chrome.runtime.sendMessage` to see examples of how this communication is performed.
+
+## The Code Flow
+
+![](https://github.com/orbit-love/orbit-browser-extension/assets/45462299/5adce40c-7e09-4f7c-ac20-474c9f6cefc1)
```

Comment:
😰 Is this a good way to show images in the README? It feels like littering the ocdebase to add the image to the repo itself. 

Solution suggested by https://stackoverflow.com/questions/14494747/how-to-add-images-to-readme-md-on-github#answer-26601810

Reply:
Yes it’s all good—no need to add the image to the repo I reckon

Reply:
Ah! Yes, you’re absolutely right!